packages/browser/src/index.ts

```diff
@@ -93,14 +98,52 @@ export default class LogtoClient {
   }
 
   protected set signInSession(logtoSignInSessionItem: Optional<LogtoSignInSessionItem>) {
-    if (!logtoSignInSessionItem) {
-      sessionStorage.removeItem(this.logtoStorageKey);
+    if (logtoSignInSessionItem) {
```

Comment:
this is not aligned too. here we use the truthy value in `if` while other props are using falsy ones.

Reply:
Updated. PTAL